foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java

```diff
@@ -58,6 +60,8 @@
     doGetHostNameAndHostAddress();
   }
 
+  static Pattern ipv6Pattern = Pattern.compile("[\\[][0-9a-fA-F:]*:[0-9a-fA-F:]*:.*[]]");
+
   private static void doGetHostNameAndHostAddress() {
     try {
       doGetIpv4AddressFromNetworkInterface();
```

Comment:
after support ipv6, we still ignore ipv6 addresses?

Reply:
Should we take ipv6 address into consideration in common situation? Or we may add a switch to let users to decide?

Reply:
@liubao68  any idea?